src/DesktopNotificationsManager.cpp

```diff
@@ -193,3 +212,86 @@ bool DesktopNotificationsManager::closeNotification(ComPtr<DesktopNotification>
     DN_LOG_ERROR("Notification " << d->getID() << " does not exist");
     return false;
 }
+
```

Comment:
This whole 🟢 block was brought from `DesktopNotification.cpp`, and then I added some changes to obtain the notification ID and send it with the callback.